readme.md

```diff
@@ -111,19 +113,20 @@ $ npm -v
 
 * **Question:** What gets logged? Why?
 
-* **Your Answer:**
+* **Your Answer:** Hello, Node! Alex. This is logged because it is exported from profile.js
+
 
 * **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
-* **Your Answer:**
+* **Your Answer:** It is a way of connecting files together. It is a function.
```

Comment:
This needs more detail.

* `module.exports` is an object and allows us to export from one file to others.
* `require` is a function and pulls in whatever was exported in another file. If nothing was exported, it imports an object.